devtools/maven/src/main/java/io/quarkus/maven/UpdateMojo.java

```diff
@@ -129,10 +130,12 @@ protected void processProjectState(QuarkusProject quarkusProject) throws MojoExe
             final QuarkusCommandOutcome result = invoker.execute();
             if (!result.isSuccess()) {
                 throw new MojoExecutionException(
-                        "The command did not succeed.");
+                        "Failed to apply the updates.");
             }
+        } catch (QuarkusUpdateExitErrorException e) {
+            throw new MojoExecutionException(e.getMessage());
```

Comment:
Shouldn't the underlying exception be propagated here?

Reply:
No, in that very case, I specifically don't want it to be propagated. The exception is completely useless, it's just a command exiting with an error code. What's important is the log that is above so not having one more stacktrace logged below helps.